lib/index.js

```diff
@@ -61,6 +61,11 @@ function processUrl(path) {
     }
   }
 
+  // number of quasis should be exactly 1 more than expressions
+  if (quasis.length === expressions.length) {
+    quasis.push(t.templateElement({ raw: "", cooked: "" }));
+  }
+
```

Comment:
IMO this is the most important change in this PR. Number of quasis should be exactly 1 more than expressions.